### PR TITLE
Schema enable option

### PR DESCRIPTION
**Pull request by danielkoek**
We are currently using the ES sink, but since we push json files without schema's to the topic and the json files might change over time we wanted to disable the schema

Just creating the PR so you guys can use it and we don't have to use a fork all the time ;)

### CODE REVIEW

**Comment by danielkoek** [comment]
[clabot:check]

**Comment by danielkoek** [comment]
[clabot:check]

File: charts/cp-kafka-connect/values.yaml [hunk 1]
```diff
@@ -24,6 +24,8 @@ servicePort: 8083
 ## ref: https://docs.confluent.io/current/connect/userguide.html#configuring-workers
 connectKeyConverter: "io.confluent.connect.avro.AvroConverter"
 connectValueConverter: "io.confluent.connect.avro.AvroConverter"
+connectKeyConverterSchemaEnable: true
```
Comment (gAmUssA) [inline charts/cp-kafka-connect/values.yaml]: can we have this disabled by default?

File: charts/cp-kafka-connect/values.yaml [hunk 2]
```diff
@@ -24,6 +24,8 @@ servicePort: 8083
...
+connectValueConverterSchemaEnable: true
```
Comment (gAmUssA) [inline charts/cp-kafka-connect/values.yaml]: and this one?

File: charts/cp-kafka-connect/values.yaml (hunk 1 above)
Comment (danielkoek) [reply charts/cp-kafka-connect/values.yaml]: Yea sure we can the only thing was that this was enabled by default atm so i kept it that way, but will make a change to make it false by default

**Comment by danielkoek** [comment]
Do i also need to rebuild the helm package then so it is the latest (the one that index.yaml uses) or do you guys auto build that?

**Pull request merged**